hyperjumptech/monika
# [Feat] Removing Axios for Native Fetch

# Monika Pull Request (PR)

## What feature/issue does this PR add

1. Implements #1248 
2. Replacing with native fetch main Monika http transactions

## How did you implement / how did you fix it

1. Remove sendHttp wrappers with native fetch

## How to test

1. `npm start -- -c test.yaml` should work
2. `npm test` should all pass
4. `monika -c https://raw.githubusercontent.com/hyperjumptech/monika/main/config_sample/config.desktop.example.yml` should be fetched properly
5. `monika --auto-update patch` should work
